assets/js/modules/analytics-4/components/widgets/TopCitiesWidget.js

```diff
@@ -52,6 +52,14 @@ function TopCitiesWidget( { Widget } ) {
 	const topcCitiesReportOptions = {
 		...dates,
 		dimensions: [ 'city' ],
+		dimensionFilters: {
+			city: {
+				filterType: 'stringFilter',
+				matchType: 'EXACT',
+				value: '(not set)',
+				notExpression: true,
+			},
+		},
```

Comment:
This is not the correct approach and doesn't follow the IB.

In fact, this filtering approach is what was specified in the original IB, and it won't meet the AC for the reasons discussed in https://github.com/google/site-kit-wp/issues/7884#issuecomment-2459450700.

Please review the IB and amend the implementation appropriately.